moby/moby
# Enable Go Modules

**- What I did**

Enabled Go Modules support. The reason why this was not possible before ("Moby uses CalVer instead of SemVer") is no longer valid since v23.0.0

**- How I did it**

Renamed `vendor.mod` to `go.mod` and `vendor.sum` to `go.sum`.

**- How to verify it**

Everything should continue to work and build as before. IDEs and other tooling now properly understand Moby dependencies.

This change also allows to enable Dependabot for Moby repository.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
